packages/quantic/force-app/main/default/lwc/headlessLoader/headlessLoader.js

```diff
@@ -49,64 +52,92 @@ const executeInitialSearch = debounce(() => {
 /**
  * Returns true if registered components are initialized, false otherwise.
  */
-const areAllComponentsInitialized = () => !window.coveoHeadless.components.find(component => component.initialized === false);
+const areAllComponentsInitialized = () =>
+  !window.coveoHeadless.components.find(
+    (component) => component.initialized === false
+  );
 
 /**
- * Returns the registered component object if it exists. 
- * @param element 
+ * Returns the registered component object if it exists.
+ * @param element
  */
-const getRegisteredComponent = (element) => window.coveoHeadless.components.find((component) => component.element === element);
+const getRegisteredComponent = (element) =>
+  window.coveoHeadless.components.find(
+    (component) => component.element === element
+  );
 
 /**
- * Loads dependencies and returns an initialized Headless engine. 
- * @param element 
+ * Loads dependencies and returns an initialized Headless engine.
+ * @param element
  */
 async function initEngine(element) {
   let engine;
   try {
     await loadScript(element, HeadlessPath + '/browser/headless.js');
     await loadScript(element, AtomicPath + '/atomic-utils.js');
-  
-    const config = CoveoHeadless.HeadlessEngine.getSampleConfiguration();
-  
+
+    /**
+     * @type {import('../../../../../../headless/dist/index').HeadlessConfigurationOptions}
```

Comment:
Is this the only way to reference the type?

Reply:
didn't manage to do otherwise

Reply:
Hmm, we should probably export this. In the mean time, you might be able to replace this with:
```js
/*
 * @type {ReturnType<CoveoHeadless['HeadlessEngine']['getSampleConfiguration']>}
 */
```
or
```js
/*
 * @type {ReturnType<import('@coveo/headless').HeadlessEngine['getSampleConfiguration']>}
 */
```

Reply:
I tried this @btaillon didn't worked.

Reply:
removed